env/sugarizer.ini

```diff
@@ -1,54 +1,57 @@
+version=1.4.0
```

Comment:
If changes in the code of this file does not affects the Assignment Feature, then reset the changes in this file.